src/infra/ssl/sslCmdRepo.go

```diff
@@ -79,58 +78,66 @@ func (repo *SslCmdRepo) ReplaceWithSelfSigned(vhostName valueObject.Fqdn) error
 	)
 }
 
-func (repo *SslCmdRepo) isDomainMappedToServer(
-	vhost valueObject.Fqdn,
+func (repo *SslCmdRepo) createOwnershipValidationMapping(
+	targetVhostName valueObject.Fqdn,
 	expectedOwnershipHash valueObject.Hash,
-) bool {
-	vhostStr := vhost.String()
-
-	rawVhostIps, err := infraHelper.RunCmd("dig", "+short", vhostStr, "@8.8.8.8")
-	if err != nil || rawVhostIps == "" {
-		rawVhostIps, err = infraHelper.RunCmd("dig", "+short", vhostStr, "@185.228.168.9")
-		if err != nil || rawVhostIps == "" {
-			return false
-		}
-	}
-
-	rawVhostIpsParts := strings.Split(rawVhostIps, "\n")
-	if len(rawVhostIpsParts) == 0 {
-		return false
-	}
+) (mappingId valueObject.MappingId, err error) {
+	path, _ := valueObject.NewMappingPath(
+		infraData.GlobalConfigs.DomainOwnershipValidationUrlPath,
+	)
+	matchPattern, _ := valueObject.NewMappingMatchPattern("equals")
+	targetType, _ := valueObject.NewMappingTargetType("inline-html")
+	httpResponseCode, _ := valueObject.NewHttpResponseCode(200)
+	targetValue, _ := valueObject.NewMappingTargetValue(
+		expectedOwnershipHash.String(), targetType,
+	)
 
-	var serverIpAddress *valueObject.IpAddress
-	for _, rawVhostIp := range rawVhostIpsParts {
-		ipAddress, err := valueObject.NewIpAddress(rawVhostIp)
-		if err != nil {
-			continue
-		}
+	inlineHtmlMapping := dto.NewCreateMapping(
+		targetVhostName,
+		path,
+		matchPattern,
+		targetType,
+		&targetValue,
+		&httpResponseCode,
+	)
 
-		serverIpAddress = &ipAddress
-		break
+	mappingCmdRepo := mappingInfra.NewMappingCmdRepo(repo.persistentDbSvc)
+	mappingId, err = mappingCmdRepo.Create(inlineHtmlMapping)
+	if err != nil {
+		return mappingId, errors.New(
+			"CreateOwnershipValidationMappingError (" +
+				targetVhostName.String() + "): " + err.Error(),
+		)
 	}
 
-	if serverIpAddress == nil {
-		return false
-	}
+	return mappingId, nil
+}
 
-	hashUrlPath := infraData.GlobalConfigs.DomainOwnershipValidationUrlPath
-	hashUrlFull := "https://" + vhostStr + hashUrlPath
-	serverIpStr := serverIpAddress.String()
+func (repo *SslCmdRepo) filterDomainsMappedToSomewhere(
```

Comment:
What? filterDomainsMappedToSomewhere? Somewhere where? That's a weird method name.